src/error.c

```diff
@@ -622,6 +623,15 @@ void AssertionFailure(void)
     ErrorReturnVoid("Assertion failure", 0, 0, "you may 'return;'");
 }
 
+// This function is called from gasman.c, when a large allocation fails
+void TooLargeAllocError(UInt bytes, const char * reason)
+{
+    // If GAP is planning to print a SyStorOverrun message, mark it as
```

Comment:
GAP cannot plan. How about this:
```suggestion
    // If a SyStorOverrun message is scheduled, mark it as already printed
```